src/app/models/welding_point.py

```diff
@@ -5,20 +5,20 @@
 
 class WeldingPoint(Base):
     id = Column(Integer, primary_key=True, index=True)
-    project_id = Column(Integer, ForeignKey("project.id"))
-    robot_id = Column(Integer, ForeignKey("robot.id"))
-    welding_order = Column(Integer, index=True)
-    name = Column(String)
+    project_id = Column(Integer, ForeignKey("project.id"), nullable=False)
+    robot_id = Column(Integer, ForeignKey("robot.id"), nullable=False)
+    welding_order = Column(Integer, index=True, nullable=False)
+    name = Column(String, nullable=False)
     description = Column(String)
-    x = Column(Float)
-    y = Column(Float)
-    z = Column(Float)
-    x_original = Column(Float)
-    y_original = Column(Float)
-    z_original = Column(Float)
-    roll = Column(Float)
-    pitch = Column(Float)
-    yaw = Column(Float)
+    x = Column(Float, nullable=False)
+    y = Column(Float, nullable=False)
+    z = Column(Float, nullable=False)
+    x_original = Column(Float, nullable=False)
+    y_original = Column(Float, nullable=False)
+    z_original = Column(Float, nullable=False)
+    roll = Column(Float, nullable=False)
+    pitch = Column(Float, nullable=False)
+    yaw = Column(Float, nullable=False)
```

Comment:
I think doing roll, pitch, yaw not nullable could result in issues later, when we add the normal vector here as well, as we probably won't get that from the Excel files.